src/Service/Lambda/src/Enum/Architecture.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace AsyncAws\Lambda\Enum;
+
+final class Architecture
+{
+    public const ARM_64 = 'arm64';
+    public const X_86_64 = 'x86_64';
```

Comment:
hat do you think about these constants @Nyholm 

- `ARM_64` and `X_86_64`
- `ARM_64` and `X86_64`
- `ARM64` and `X86_64`

Reply:
I’m fine with all :)
Mostly because I’m lazy. 

But your third option is what looks best.